docs/organizations/managing-people.md

```diff
@@ -11,8 +11,9 @@ For **private repositories**, Codacy only analyzes commits from people in your C
 
 To list and manage the people in your organization, open your organization **Settings**, page **People**. This page also shows their last activity on Codacy.
 
-!!! note
-    In some Enterprise plans, Codacy automatically assigns a seat to any person with at least one analyzed commit to your private repositories over the past 90 days and frees up the seat after 90 days of code analysis inactivity.
+!!! note "In some Enterprise plans"
+    - Codacy automatically assigns a seat to any person with at least one analyzed commit to your private repositories over the past 90 days and frees up the seat after 90 days of code analysis inactivity.
+    - **Members** are people that signup and joined an Organization in Codacy, and **Authors** are people that contribuit to code changes. It can happen that Authors are not Codacy Members. 
```

Comment:
```suggestion
    - **Members** are people that signup and joined an Organization in Codacy, and **Authors** are people that contribute to code changes. It can happen that Authors are not Codacy Members. 
```